hs_app_netCDF/models.py

```diff
@@ -51,18 +51,24 @@ def create(cls, **kwargs):
 
         :param kwargs: the 'value' in kwargs should be a dictionary
         """
-
+        value_arg_dict = None
         if 'value' in kwargs:
+            value_arg_dict = kwargs['value']
+        elif '_value' in kwargs:
+            value_arg_dict = json.loads(kwargs['_value'])
```

Comment:
@ironfroggy the _value parameter being passed in is generated from django.forms.models.model_to_dict() which converts model values to dictionaries. Will "JSON serialized unicode string dictionary" be clear enough? If not, could you suggest some language I can put in to docstring to better explain this parameter?